ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml

```diff
@@ -1131,7 +1131,7 @@ presubmits:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-azure-upi,?($|\s.*)
   - agent: kubernetes
-    always_run: false
+    always_run: true
```

Comment:
Is this an intentional change?

Reply:
This will be reverted. This forces the rehearsal on master branch.